nncf/common/graph/graph.py

```diff
@@ -673,3 +673,18 @@ def get_all_edges(self) -> Generator[NNCFGraphEdge, None, None]:
         for nx_edge in self._nx_graph.edges:
             yield self.get_edge(self.get_node_by_key(nx_edge[0]),
                                 self.get_node_by_key(nx_edge[1]))
+
+    def remove_nodes(self, nodes: List[NNCFNode]) -> None:
```

Comment:
The graph topology of NNCFGraph is immutable. Could you shortly explain why do you need this method.

Reply:
This was redundant code. Removed this method